webservice/template/assignment/submission.tmpl

```diff
@@ -1,49 +1,118 @@
+                                    <textarea name="{{ .Name }}" id="{{ .Name }}" class="form-control" disabled></textarea>
+                                {{ end }}
+
+                            </div>
+                        {{ end }}
                     {{ end }}
-                {{ end }}
+                </div>
             </div>
+            {{if .IsTeacher}}
+                <div class="col-12 mb-3">
+                    {{range .MyReviews}}
+                        <div class="card mb-3">
+                            <div class="card-header bg-secondary">
+                                <h5 class="card-title mb-0">Review by {{.Reviewer}}</h5><!-- TODO (Svein): Plugin for converting id to username -->
+                            </div>
+                            <div class="card-body">
+                                {{range .Answers}}
+                                    <div class="form-group">
+                                        <label for="">{{.Label}}</label>
+                                        {{if eq .Type "text"}}
+                                            <input type="text" class="form-control" value="{{.Answer}}" disabled>
+                                        {{end}}
+                                    </div>
+                                {{end}}
+                            </div>
+                        </div>
+                    {{end}}
+                </div>
+            {{else}}
+                {{if .Delivered}}
+                    <div class="col-12">
+                        <h2 class="display-4">Peer Review</h2>
+                        <hr>
+                        <form action="/assignment/{{.Assignment.ID}}/submission/{{.User.ID}}" method="post">
+                            {{ range .Review.Form.Fields }}
+                                {{if eq .Type "radio"}}
+                                    {{$R := .}}
+                                    {{$V := 1}}
+                                    <fieldset>
+                                        <legend>{{$R.Label}}</legend>
+                                    {{range SPLIT_CHOICES .Choices}}
+                                        <div class="form-check form-check-inline">
+                                            <input class="form-check-input" type="radio" name="{{$R.Name}}" id="{{$R.Name}}" value="{{$V}}">
+                                            <label class="form-check-label" for="{{$R.Name}}">{{.}}</label>
+                                        </div>
+                                        {{INCREMENT $V}}
+                                    {{end}}
+                                    </fieldset>
+
+                                {{else}}
+                                <div class="form-group{{if eq .Type "checkbox"}} form-check{{end}}">
+                                    {{if ne .Type "checbox"}}<label for="{{ .Name }}">{{ .Label }}</label>{{end}}
```

Comment:
If you are showing all types here, then you should maybe show all types the other places it's used